test/utils/PathUtils.test.ts

```diff
@@ -356,4 +359,256 @@ describe('PathUtils', () => {
       });
     });
   });
+
+  describe('#collectValuesAtElementIdOrPath', () => {
+    let object: InstanceDefinition;
+    beforeEach(() => {
+      const json = readJSONSync(
+        path.join(
+          __dirname,
+          '..',
+          'testhelpers',
+          'testdefs',
+          'r4-definitions',
+          'package',
+          'Observation-20minute-apgar-score.json'
+        )
+      );
+      object = InstanceDefinition.fromJSON(json);
+    });
+
+    it('should collect simple singular properties with resourceType prefix and resourceType', () => {
+      const results = collectValuesAtElementIdOrPath('Observation.status', object);
+      expect(results.values).toEqual(['final']);
+      expect(results.ignoredSliceRequirements).toEqual([]);
+    });
+
+    it('should collect simple singular properties with resourceType prefix and no resourceType', () => {
+      //@ts-ignore
```

Comment:
I don't think this usage of `@ts-ignore` or the following uses are necessary. I was able to remove them and run the tests without any problems.

Reply:
I added these because VS Code was flagging them for me. It turns out that VS Code tends to behave better if you put a test-specific tsconfig straight into the `test` folder. Other TypeScript projects I work on have done this as well, so there is precedent. After doing this, I was able to remove the `//@ts-ignore` statements.  See 302970c.